### PR TITLE
Adds article from newsletter jan 22

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
## Motivation

Have a record of our newsletter

## Approach

Set the newsletter on markdown, and I'll link from the email to here.

[Rendered preview](https://deploy-preview-184--frontside.netlify.app/blog/2022-01-26-what-is-dx/)

### CODE REVIEW

File: src/blog/2022-01-26-what-is-dx.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+templateKey: blog-post
+title: >-
+  What is DX anyways: a chronoloical overview
```
Comment (minkimcello) [inline src/blog/2022-01-26-what-is-dx.md]: `anyways` ➡️ `anyway`
`chronoloical` ➡️ `chronological`

File: src/blog/2022-01-26-what-is-dx.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+author: Jorge Lainfiesta
+description: >-
+  You can only optimize for concrete indicators, that’s why having a clear understanding of DX is important to improve it. In this article, we review the main trends of the past decade.
+tags:
+  - dx
+  - newsletter
+img: /img/2022-what-is-dx.png
+---
+
+It was bound to happen: after UX (user experience) and CX (customer experience) it was only a matter of time before DX—Developer Experience—took center stage. Nowadays it's in the marketing copy of every library and product aimed at developers. But what does DX mean?
```
Comment (minkimcello) [inline src/blog/2022-01-26-what-is-dx.md]: > DX-Developer Experience-

why not `DX (developer experience)` for consistency?

File: src/blog/2022-01-26-what-is-dx.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+img: /img/2022-what-is-dx.png
+---
+
+It was bound to happen: after UX (user experience) and CX (customer experience) it was only a matter of time before DX—Developer Experience—took center stage. Nowadays it's in the marketing copy of every library and product aimed at developers. But what does DX mean? 
+
+As it turns out, the definition of DX has evolved in important ways over the past decade. This month in the DX Newsletter, we’ll review three trends found in academic and industry articles to set the stage for a broader conversation.
+
+## 1. The first traces of DX 
+
+The combination of “Developer” and “experience” yields a jumble of ambiguous concepts throughout the years. For instance, most results point to the seniority of a developer.
```
Comment (minkimcello) [inline src/blog/2022-01-26-what-is-dx.md]: I think you should either lower case `"Developer"` or upper case `"experience"` depending on how you want to emphasize it

File: src/blog/2022-01-26-what-is-dx.md (hunk 2 above)
Comment (jorgelainfiesta) [reply src/blog/2022-01-26-what-is-dx.md]: I think it was some sort of way of making emphasis, but you're right

File: src/blog/2022-01-26-what-is-dx.md (hunk 1 above)
Comment (jorgelainfiesta) [reply src/blog/2022-01-26-what-is-dx.md]: thanks!!

File: src/blog/2022-01-26-what-is-dx.md (hunk 3 above)
Comment (jorgelainfiesta) [reply src/blog/2022-01-26-what-is-dx.md]: Good point, thanks!

**Review by cowboyd — APPROVED** [review]
I read this in the newsletter, and loved it. So seems like it's a no-brainer for the website

**Pull request merged**